app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -964,4 +989,16 @@ private Intent createChooserIntent(WebChromeClient.FileChooserParams params) {
             return FilePickerUtil.getFilePickerIntent(getActivity(), title, mimeTypes);
         }
     }
+
+    @Override
+    public void onScreenshotTaken(String screenshotPath, String title) {
```

Comment:
My personal feeling is declaring a new type in FragmentListener.TYPE.SHOW_SCREENSHOT_HINT will be cleaner then current implementation.

We won't need a lot of getActivity() in this case, and another benefit would be we can collect all Dialog showing logic there (a simple call to DialogUtils.showScreenshotOnBoardingDialog(this);)

I'm particularly against this because everytime we call getActivity() it can suddenly return null. Keep all logic in the activity seems just more right to me. I'm not 100% certain on this so let me know if you think otherwise.